src/index.js

```diff
@@ -1,42 +1,45 @@
 import './style.css';
+import './modules/checkbox';
 
 const list = document.querySelector('.todo-list');
 
-const myList = [
-  {
-    description: 'wash car',
-    completed: false,
-    index: 1,
-  },
-  {
-    description: 'homework',
-    completed: false,
-    index: 2,
-  },
-  {
-    description: 'gym',
-    completed: false,
-    index: 3,
-  },
-  {
-    description: 'study rails',
-    completed: false,
-    index: 4,
-  },
-];
+let myList = [];
+
+if (localStorage.myList !== undefined) {
+  myList = JSON.parse(localStorage.myList);
+}
```

Comment:
- [ ] in this milestone, you are still supposed to keep the array of todo's you added in the first milestone. When I start your script, those array items should be rendered. 
- [ ] The only thing you add is an event listener on the checkbox (change). When I check the box, the status should be updated in the local storage as well so the user gets the correct list values when the page reloads. I hope this explanation is clear enough.